.github/workflows/prepare-release.yml

```
@@ -0,0 +1,55 @@
+name: Prepare release
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: new release version
+        required: true
+        default: (for example, 0.10.5)
+
+jobs:
+  release:
+    name: Prepare release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.4
+
+      - name: Check input
+        run: |
+          if [[ ! "${{ github.event.inputs.release_version }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo 'version "${{ github.event.inputs.release_version }}" not in ###.###.### format'
+            exit 1
+          fi
+          if [[ $( git ls-remote --tags origin refs/tags/v${{ github.event.inputs.release_version }} ) ]]; then
+            echo 'Tag "v${{ github.event.inputs.release_version }}" already exists. Double-check the version to release!'
+            exit 1
+          fi
+
+      - name: Create tag for release and prepare branch for next development version
+        run: |
+          git checkout -b release-v${{ github.event.inputs.release_version }}
+          git config user.email ${{ github.actor }}@users.noreply.github.com
+          git config user.name ${{ github.actor }}
+
+          ./mvnw --batch-mode versions:set versions:commit -DnewVersion=${{ github.event.inputs.release_version }}
+          git commit -am "preparing release ${{ github.event.inputs.release_version }}"
+          git tag v${{ github.event.inputs.release_version }}
+
+          ./mvnw --batch-mode org.apache.maven.plugins:maven-release-plugin:2.5.3:update-versions
+          git commit -am "next development version"
+
+          git push origin v${{ github.event.inputs.release_version }}
+          git push origin release-v${{ github.event.inputs.release_version }}
+
+      - name: Create pull request
+        uses: repo-sync/pull-request@v2.6
+        id: create-pr
+        with:
+          # Use a personal token to file a PR as a non-bot author to trigger other workflows (e.g., unit tests):
```

This comment does not seem applicable here.

---

The comment applies to the following line `github_token: ...`. Note that for other steps, it is a bot author, but for this one, we use a personal token (`secrets.GA_RELEASE_PR_PERSONAL_TOKEN`).

---

That makes sense. I think I was confused "as a non-bot author" part.

```suggestion
          # Use a personal token to file a PR to trigger other workflows (e.g., unit tests):
```

---

To me, "as a non-bot author" is correct (the PR will seen as filed by a user like [this](https://github.com/chanseokoh/app-gradle-plugin/pull/3)), but I have no problem just removing it.